src/Quartz/CronExpression.cs

```diff
@@ -208,12 +208,21 @@ namespace Quartz
     /// <author>Contributions from Mads Henderson</author>
     /// <author>Refactoring from CronTrigger to CronExpression by Aaron Craven</author>
     /// <author>Marko Lahma (.NET)</author>
+#if BINARY_SERIALIZATION
     [Serializable]
-    public class CronExpression : ICloneable, IDeserializationCallback
+#endif // BINARY_SERIALIZATION
+    [DataContract]
+    public class CronExpression : object
+#if ICLONEABLE
```

Comment:
ICloneable might be something that we could just drop. We can still offer the Clone method that could return the actual type and state clearly whether it's a deep or a shallow copy.